docs/wp/ipc/index.md

```diff
@@ -462,7 +462,7 @@ c  | 43.7764
 
 ### Broadcast 
 
-Much of the overhead of sending a message via IPC is in serializing the data before sending. It is possible to ‘async broadcast’ the same message to multiple handles using the internal `-25!` function. This will serialize the message once and send to all handles to reduce CPU and memory load. An error in publishing to any handle will result in the message not being sent to any of the handles, regardless of the handle’s position in the list.
+Much of the overhead of sending a message via IPC is in serializing the data before sending. It is possible to ‘async broadcast’ the same message to multiple handles using the internal [`-25!`](../../basics/internal.md#-25x-async-broadcast) function. This will serialize the message once and send to all handles to reduce CPU and memory load. An error in publishing to any handle will result in the message not being sent to any of the handles, regardless of the handle’s position in the list.

```

Comment:
Much of the overhead of sending a message via IPC is in serializing the data before sending. It is possible to ‘async broadcast’ the same message to multiple handles using the internal [`-25!`](../../basics/internal.md#-25x-async-broadcast) function. This serializes the message once and sends to all handles to reduce CPU and memory load. An error in publishing to any handle results in the message not being sent to any of the handles, regardless of the handle’s position in the list.

Reply:
updated